### PR TITLE
Disable legacy sequence parsing in RA.

### PR DESCRIPTION
**Pull request by pchote**
Like #7738, but for RA.  Only two more PRs after this one!

### CODE REVIEW

File: mods/ra/sequences/misc.yaml [hunk 1]
```diff
@@ -206,7 +210,9 @@ moveflsh:
 		Start: 0
 		Length: *
 		Tick: 80
-
+		UseTilesetExtension: true
+		TilesetOverrides:
+			DESERT: TEMPERAT
 select:
```
Comment (penev92) [inline mods/ra/sequences/misc.yaml]: New line above here?

Comment (reaperrr) [inline mods/ra/sequences/misc.yaml]: Agreed, that line removal looks wrong.

File: mods/ra/sequences/decorations.yaml [hunk 2]
```diff
@@ -1,4 +1,10 @@
 tc04:
+	Defaults:
+		UseTilesetExtension: true
+		TilesetOverrides:
+			DESERT: TEMPERAT
+			INTERIOR: TEMPERAT
+
```
Comment (reaperrr) [inline mods/ra/sequences/decorations.yaml]: This empty line needs to be removed.

File: mods/ra/sequences/decorations.yaml [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
 	idle:
 		Start: 0
```
Comment (reaperrr) [inline mods/ra/sequences/decorations.yaml]: All the ```Start: 0``` from here...

File: mods/ra/sequences/decorations.yaml [hunk 4]
```diff
@@ -449,211 +647,239 @@ brl3:
...
-	idle:
+	idle: boxes08.int
 		Start: 0
 		Length: *
+		AddExtension: false
 
 boxes09:
-	idle:
+	idle: boxes09.int
 		Start: 0
```
Comment (reaperrr) [inline mods/ra/sequences/decorations.yaml]: ...to here can be removed.

File: mods/ra/sequences/infantry.yaml [hunk 5]
```diff
@@ -60,11 +60,19 @@ e1:
 	die6: electro
 		Start: 0
```
Comment (reaperrr) [inline mods/ra/sequences/infantry.yaml]: I think ```Start: 0``` can be removed throughout this file.

File: mods/ra/sequences/misc.yaml [hunk 6]
```diff
@@ -76,6 +76,10 @@ explosion:
 		Start: 0
```
Comment (reaperrr) [inline mods/ra/sequences/misc.yaml]: All the ```Start: 0``` in this file can go, too.

File: mods/ra/sequences/structures.yaml [hunk 7]
```diff
@@ -9,6 +9,9 @@ fcom:
 	bib: bib3
 		Start: 0
```
Comment (reaperrr) [inline mods/ra/sequences/structures.yaml]: For consistency and cleanup, the ```Start: 0``` should go from this file, too, while you're at it.

**Comment by pchote** [comment]
Added fixes by @reaperrr.

**Comment by obrakmann** [comment]
Looks ok to me. Played games on all tilesets except interior, tested the editor on all tilesets. :+1:

**Comment by reaperrr** [comment]
Tested and looks good to me.

There are two minor hiccups remaining, I forgot to add that missing newline @penev92 pointed out and noticed that the tent no longer uses tileset-specific art.
But since there's currently no ```tent.tem``` the latter is not that quick to fix, therefore I'd rather deal with these in a follow-up myself so this pr's merge doesn't get delayed, in other words :+1:

**Comment by Mailaender** [comment]
This also broke the palette for walls in the DESERT tileset in the editor:

image

**Comment by Mailaender** [comment]
Also broke resource placement on DESERT maps using the WinForms editor:

image

**Comment by reaperrr** [comment]
I'll add the fixes to #7747.

**Comment by reaperrr** [comment]
On second thought, the cleanest fix would probably be to make the editor ```TilesetOverrides```-aware.

**Comment by pchote** [comment]
@reaperrr: the whole problem here is that the editor doesn't understand or use sequences at all. It uses a collection of hacks to find a .shp/.tem/etc with the same filename as the actor.  I will adjust #7749 to fix some of these issues.

**Pull request merged**